host_guest_description.md

```diff
@@ -49,16 +49,14 @@ It is also worth emphasizing that some of these compounds, particularly quinine,
 We have endeavored to provide correct starting structures, but would advise checking our work, and keeping an eye on the stereochemistry during all cheminformatics and modeling steps, to avoid inadvertent changes in stereochemistry.
 
 CB8 has a larger binding cavity than CB7, and there is the possibility of 1:1 and/or 1:2 CB[8]:guest binding.
-This has been assessed experimentally and aricept and cyclohexane diamine were found bind in 1:2 complexes, whereas the other compounds do not.
-Oxaliplatin is also potentially complex due to the platinum atom it contains.
+This has been assessed experimentally and aricept (CB8-G11) and cyclohexane diamine (CB8-G12) were found bind in 1:2 complexes, whereas the other compounds do not.
+Oxaliplatin (CB8-G13) is also potentially complex due to the platinum atom it contains.
```

Comment:
Taking a quick look at the mol2 files, I believe aricept is CB8-G12 and cyclohexane diamine is CB8-G11.

Reply:
Geez, yes. I copied my own notes wrong!!